1890rules-en-translation.tex

```diff
@@ -905,7 +905,7 @@ \subsection{Minor companies and public companies}
 \item Place a new station token. (Optional, public company only)
 \item Run trains for revenue.
 \item Public companies pay dividends or withhold. Minor companies will
-  pay dividends.
+  pay 50% dividends.
```

Comment:
The % sign needs to be escaped with a backslash, otherwise LaTeX sees it as the start of a comment